From c6c1c710541c9b90dafa3e2095c2385e499cdc8a Mon Sep 17 00:00:00 2001
From: Alan Viverette <alanv@google.com>
Date: Fri, 21 Apr 2017 10:47:41 -0400
Subject: [PATCH 11/67] Use explicit support library prebuilts

Test: make Gallery2
Test: make ANDROID_BUILDSPEC=vendor/google/build/app_build_spec.mk
Change-Id: I59208256a3abd71ef8b352b132a4de0d61d88c03
---
 Android.mk | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/Android.mk b/Android.mk
index 58ad91236..31668fd28 100755
--- a/Android.mk
+++ b/Android.mk
@@ -6,7 +6,7 @@ LOCAL_MODULE_TAGS := optional
 
 LOCAL_JAVA_LIBRARIES := telephony-common
 
-LOCAL_STATIC_JAVA_LIBRARIES := android-support-v13
+LOCAL_STATIC_JAVA_LIBRARIES := android-support-v13-prebuilt
 LOCAL_STATIC_JAVA_LIBRARIES += org.codeaurora.gallery.common
 LOCAL_STATIC_JAVA_LIBRARIES += xmp_toolkit
 LOCAL_STATIC_JAVA_LIBRARIES += mp4parser
@@ -30,7 +30,7 @@ LOCAL_SRC_FILES += $(call all-java-files-under, src_pd)
 
 # $(warning Anand Commend LOCAL_RENDERSCRIPT_SKIP_INSTALL is $(LOCAL_RENDERSCRIPT_SKIP_INSTALL))
 
-LOCAL_RESOURCE_DIR += $(LOCAL_PATH)/res $(LOCAL_PATH)/../../../$(SUPPORT_LIBRARY_ROOT)/compat/res
+LOCAL_RESOURCE_DIR += $(LOCAL_PATH)/res
 
 LOCAL_AAPT_FLAGS := --auto-add-overlay
 LOCAL_AAPT_FLAGS += --extra-packages android.support.compat
-- 
2.17.1

